src/asleep/get_sleep.py

```diff
@@ -148,6 +148,20 @@ def transform_data2model_input(
         non_wear = np.load(non_wear_path)
     return data2model, times, non_wear
 
+def mean_temp_and_light(data):
+    # Calculates mean temperature and light for each 30s window
+    print("Calculating mean temperature and light for each 30s interval.")
+    
+    data['time'] = pd.to_datetime(data['time'])
+    # Grouping by time
+    grouped = data.groupby(pd.Grouper(key='time', freq='30S', origin=data['time'].min()))
+    # Calculating the mean temperature and light for each group
+    mean_values = grouped[['temperature', 'light']].mean().iloc[:-1]
```

Comment:
Could you include an expiation handler here to exit the file processing when no temperature or light data is available? No all the devices have these information.

Reply:
Also some of the CI testes failed. Would be good if these are also addressed :D 

https://github.com/OxWearables/asleep/actions/runs/9999071709/job/27810403819?pr=53

Reply:
@spiousas the CI test still failed though. The formatting should be easy to fix :D

Reply:
I am sorry, I missed some fixes. Now it should be OK.